README.md

```diff
@@ -1,32 +1,34 @@
 Magento Extension for Dropday
-===============
+This is a Magento extension that enables Dropday functionality for your Magento store.
 
-Dropday.io is a service to automate your webshop orders. With Dropday.io you can forward orders automatically to your suppliers via different methods, for example by placing a XML file on FTP, sending an email or letting a Dropday employee place a manual order on another webshop. This solution is ideal for dropshipping, fulfillment or other types or supplier automation.
+Installation
+Create a directory named sites on your system:
 
-This extension will connect to the API and forward your orders to Dropday.io. To connect to the API make an account at [Dropday.io](https://dropday.io/register).
+sudo mkdir sites
 
-## How to install the extension?
+Navigate to the sites directory:
 
-### 1. Installion via Composer
+cd sites
 
-```
-composer require dropday-io/module-orderautomation
-```
+Run the following command to execute the installation scripts and set up Docker:
 
-### 2. Enable
+curl -s https://raw.githubusercontent.com/devopsoptimbytes/docker-magento/master/lib/onelinesetup | bash -s -- magento.test 2.4.6-p2 community
 
-```
-php bin/magento module:enable Dropday_OrderAutomation
-php bin/magento setup:upgrade
-php bin/magento cache:clean
-```
+During the process, you will be prompted for credentials for repo.magento.com:
 
-### 3. Configure
+Username: 217ab4e2c17a820d861e3f917785e066
+Password: 0497283a6f53ed3c3358752151ff2bfc
 
-Stores &rarr; Configuration &rarr; Dropday &rarr; Order Automation
 
-- General Settings
-   - Enabled - Enable Order Automation
-   - Test Mode - Enable Test mode
-   - Account ID - Account ID from Dropday Dashboard
-   - API Key - Secret Key be provided by Dropday Dashboard
+After the installation is complete, create the credentials for the Magento admin dashboard using the following command:
+
+
+bin/magento admin:user:create --admin-user='username' --admin-password='password123' --admin-email='user@example.com' --admin-firstname='FirstName' --admin-lastname='LastName'
+
+
+Once the admin user is created, disable the following modules using the following command:
+
+bin/magento module:disable Magento_AdminAdobeImsTwoFactorAuth Magento_TwoFactorAuth
+
+
+This will disable the Magento_AdminAdobeImsTwoFactorAuth and Magento_TwoFactorAuth modules.
```

Comment:
You just replaced all other information about this open source project, why?

Put it back and make subsection, with installation instructions. [See me previous review.](https://github.com/dropday-io/Magento/pull/18#discussion_r1356599232).